app/models/validations/soft_validations.rb

```diff
@@ -188,6 +191,12 @@ def reasonother_might_be_existing_category?
     PHRASES_LIKELY_TO_INDICATE_EXISTING_REASON_CATEGORY_REGEX.match?(reasonother)
   end
 
+  def multiple_partners?
+    return unless hhmemb
+
+    (2..hhmemb).count { |n| public_send("relat#{n}") == "P" } > 1
```

Comment:
suggest https://apidock.com/rails/Enumerable/many%3F here, up to you :)

Reply:
Ah, of course, tyty!